apps/src/BlockStaticAnalyzer.js

```diff
@@ -0,0 +1,408 @@
+/* jshint
+strict: true,
+curly: true,
+funcscope: true,
+newcap: true,
+nonew: true,
+shadow: false,
+unused: true,
+
+maxlen: 90,
+maxparams: 3,
+maxstatements: 200
+*/
+/* global -Blockly */
+'use strict';
+
+var constants = require('./constants');
+var TestResults = constants.TestResults;
+
+/**
+ * @class BlockStaticAnalyzer
+ *
+ * Answers questions about a user's solution within a particular blockSpace
+ *
+ * @param {Blockly} blockly The blockly instance being examined
+ */
+var BlockStaticAnalyzer = function ( blockly ) {
```

Comment:
Not sure if we have an explicit rule about this one way or another, but we don't usually have whitespace between the parens and the params. i.e. function (blockly) instead of function ( blockly )